app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationCollection/integrationCollection.jsx

```diff
@@ -29,9 +29,9 @@ export const IntegrationCollection = ({ items, disabled, openIntegration }) => {
         {items.map((item) => (
           <IntegrationCollectionItem
             key={item.id}
+            item={item}
             id={item.id}
             disabled={disabled}
-            data={item}
             title={item.name}
```

Comment:
If we describe items : PropTypes.array.isRequerd.
Please remove it from default props.